region/crateria/west/Gauntlet Entrance.json

```diff
@@ -80,6 +80,103 @@
     }
   ],
   "strats": [
+    {
+      "link": [1, 1],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "canTrickyJump",
+        "canDodgeWhileShooting",
+        {"or": [
+          "ScrewAttack",
+          "h_useMorphBombs",
+          {"and": [
+            "Morph",
+            {"or": [
+              {"ammo": {"type": "PowerBomb", "count": 3}},
+              {"and":[
+                "h_CrystalFlash",
+                {"ammo": {"type": "PowerBomb", "count": 2}}
+              ]}
+            ]}
+          ]}
+        ]},
+        {"or": [
+          {"resourceAvailable": [{"type": "ReserveEnergy", "count": 1}]},
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"resourceMissingAtMost": [{"type": "Missile", "count": 0}]},
+            {"resourceMissingAtMost": [{"type": "Super", "count": 0}]},
+            {"partialRefill": {"type": "ReserveEnergy", "limit": 20}}
+          ]}
+        ]},
+        {"or": [
+          {"canShineCharge": {"usedTiles": 18, "steepUpTiles": 2, "steepDownTiles": 1, "startingDownTiles": 0, "openEnd": 1}},
+          {"and":[
+            {"or": [
+              "canUseFrozenEnemies",
+              {"ammo": {"type": "Super", "count": 1}}
+            ]},
+            {"canShineCharge": {"usedTiles": 21, "steepUpTiles": 2, "steepDownTiles": 2, "startingDownTiles": 1, "openEnd": 0}}
+          ]}
+        ]},
+        {"autoReserveTrigger": {"maxReserveEnergy": 95}},
+        "canRModeSparkInterrupt"
+      ],
+      "clearsObstacles": ["A"],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "note": [
+        "Clear the bomb barriers and farm two Wavers. You can keep one to interrupt, but if you need to",
+        "farm it, you can still use the acid to interrupt.",
+        "If you mess up the spark position, remember to keep at least any button held to prevent a Yapping Maw crash."
+      ]
+    },
+    {
+      "link": [1, 1],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit) - Yapping Maw Kill",
```

Comment:
This isn't logically unique enough from the other strat so there is no reason to include it.